src/HotChocolate/Core/test/Types.Tests/Internal/ExtendedTypeTests.cs

```diff
@@ -301,6 +301,7 @@ public void IsEqual_Object_Byte_String_False()
         [InlineData(typeof(IEnumerable<string>), "IEnumerable<String>", "String")]
         [InlineData(typeof(IReadOnlyCollection<string>), "IReadOnlyCollection<String>", "String")]
         [InlineData(typeof(IReadOnlyList<string>), "IReadOnlyList<String>", "String")]
+        [InlineData(typeof(IQuery<string>), "IQuery<String>", "String")]
```

Comment:
[InlineData(typeof(IExecutable<string>), "IExecutable<String>", "String")]

Reply:
that is something we cannot allow
we need to know the type for the list :)